packages/ui/src/theme/css/component/select.scss

```diff
@@ -30,6 +30,11 @@
     }
   }
 
+  // For Chrome on Windows, fixes darkmode text color
+  option {
```

Comment:
My only question is whether we want to namespace this inside of our own class? This could change the default for other library code the app developer may be using?

Reply:
Hm do you mean making this a flat `.amplify-select__option {}`? or something else? Currently it's `.amplify-select option {}`. 

I thought about that first option but our SelectField allows passing native `<option>` elements as [children](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/src/primitives/SelectField/SelectField.tsx#L28), so I think we'd have to do some fancy iterating on `children` to place this class name on them (and making sure we're not removing any classes/attributes a customer might customize on their `<option>`s)